rcl_action/include/rcl_action/impl/action_client.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCL_ACTION__IMPL__ACTION_CLIENT_H_
+#define RCL_ACTION__IMPL__ACTION_CLIENT_H_
+
+#include "rcl_action/types.h"
+#include "rcl/rcl.h"
+
+typedef struct rcl_action_client_impl_t
```

Comment:
Is it possible to move the implementation headers to the `src` directory? This way we avoid installing them and having users misuse them.

If not, we can leave it as-is.

Reply:
Moved